terracumber_config/tf_files/MLM-5.1-build-validation-NUE.tf

```diff
@@ -0,0 +1,1213 @@
+  type = string
+}
+
+variable "SCC_PASSWORD" {
+  type = string
+}
+
+variable "CONTAINER_REPOSITORY" {
+  type = string
+}
+
+variable "GIT_USER" {
+  type = string
+  default = null // Not needed for master, as it is public
+}
+
+variable "GIT_PASSWORD" {
+  type = string
+  default = null // Not needed for master, as it is public
+}
+
+variable "ZVM_ADMIN_TOKEN" {
+  type = string
+}
+
+terraform {
+  required_version = "1.0.10"
+  required_providers {
+    libvirt = {
+      source = "dmacvicar/libvirt"
+      version = "0.8.1"
+    }
+    feilong = {
+      source = "bischoff/feilong"
+      version = "0.0.6"
+    }
+  }
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://suma-06.mgr.suse.de/system"
+}
+
+provider "libvirt" {
+  alias = "suma-arm"
+  uri = "qemu+tcp://suma-arm.mgr.suse.de/system"
+}
+
+provider "feilong" {
+  connector   = "https://feilong.mgr.suse.de"
+  admin_token = var.ZVM_ADMIN_TOKEN
+  local_user  = "jenkins@jenkins-worker.mgr.suse.de"
+}
+
+module "base_core" {
+  source            = "./modules/base"
+
+  cc_username       = var.SCC_USER
+  cc_password       = var.SCC_PASSWORD
+  product_version   = "head"
+  name_prefix       = "mlm-bv-51-"
+  use_avahi         = false
+  domain            = "mgr.suse.de"
+  images            = [ "sles12sp5o", "sles15sp2o", "sles15sp3o", "sles15sp4o", "sles15sp5o", "sles15sp6o", "slemicro51-ign", "slemicro52-ign", "slemicro53-ign", "slemicro54-ign", "slemicro55o", "slmicro60o", "slmicro61o", "almalinux8o", "almalinux9o", "centos7o", "libertylinux9o", "oraclelinux9o", "rocky8o", "rocky9o", "ubuntu2204o", "ubuntu2404o", "debian12o", "opensuse155o", "opensuse156armo" ]
+
+  mirror            = "minima-mirror-ci-bv.mgr.suse.de"
+  use_mirror_images = true
+
+  testsuite         = true
+
+  provider_settings = {
+    pool        = "ssd"
+    bridge      = "br0"
+    additional_network = "192.168.50.0/24"
+  }
+}
+
+module "base_arm" {
+  providers = {
+    libvirt = libvirt.suma-arm
+  }
+
+  source            = "./modules/base"
+
+  cc_username       = var.SCC_USER
+  cc_password       = var.SCC_PASSWORD
+  product_version   = "head"
+  name_prefix       = "mlm-bv-51-"
+  use_avahi         = false
+  domain            = "mgr.suse.de"
+  images            = [ "opensuse156armo" ]
+
+  mirror            = "minima-mirror-ci-bv.mgr.suse.de"
+  use_mirror_images = true
+
+  testsuite         = true
+
+  provider_settings = {
+    pool        = "ssd"
+    bridge      = "br0"
+  }
+}
+
+module "base_s390" {
+  source            = "./backend_modules/feilong/base"
+
+  name_prefix       = "mlm-bv-51-"
+  domain            = "mgr.suse.de"
+  product_version   = "head"
+
+  testsuite         = true
+}
+
+module "server_containerized" {
+  source             = "./modules/server_containerized"
+  base_configuration = module.base_core.configuration
+  name               = "server"
+  image              = "slemicro55o"
+  provider_settings = {
+    mac                = "aa:b2:92:42:01:01"
```

Comment:
Did not check the MAC addresses, it will be good to link the MR about it on this PR.

Reply:
Both MRs are linked in the first post. See
[gitlab.suse.de/galaxy/infrastructure/-/merge_requests/1012](https://gitlab.suse.de/galaxy/infrastructure/-/merge_requests/1012)
[gitlab.suse.de/galaxy/infrastructure/-/merge_requests/1014](https://gitlab.suse.de/galaxy/infrastructure/-/merge_requests/1014)